service/src/main/java/bio/terra/workspace/service/spendprofile/client/BpmClientProvider.java

```diff
@@ -0,0 +1,9 @@
+package bio.terra.workspace.service.spendprofile.client;
+
+import bio.terra.profile.api.ProfileApi;
+import bio.terra.workspace.service.iam.AuthenticatedUserRequest;
+
+/** Knows how to instantiate Billing Profile Manager clients */
+public interface BpmClientProvider {
```

Comment:
Nice pattern -- is this so the client can be more easily mocked for tests?

Wonder if other clients are following this approach..

Reply:
One of the goals of the _____Service modules is to wrap the interaction with the external service, so this seems like an unnecessary layer.